pt-BR/form_helpers.md

```diff
@@ -991,7 +991,7 @@ As a convenience you can instead pass the symbol `:all_blank` which will create
 
 Rather than rendering multiple sets of fields ahead of time you may wish to add them only when a user clicks on an 'Add new address' button. Rails does not provide any built-in support for this. When generating new sets of fields you must ensure the key of the associated array is unique - the current JavaScript date (milliseconds since the [epoch](https://en.wikipedia.org/wiki/Unix_time)) is a common choice.
 
-Using form_for and form_tag
+Usando _form_for_ e _form_tag_
```

Comment:
Como no guia em pt-BR a gente usa o parameterize pra gerar os links, se você quiser pode até colocar como notação de `código` no titulo.

Essa é mais uma dica dos internals. Mas não tem problema ficar em itálico, fique a vontade para mudar ou não 😄

Reply:
Acho legal seguir a convenção, se é usar a `tag de código` acho interessante mudar então. Valeu! =)